
pmx does not compile with musl libc because the included
libf2c uses the nonstandard type __off64_t. The below patch,
which is adapted from current libf2c, fixes this.

Felix Janda [mailto:felix.janda@posteo.de]


--- a/pmx-2.6.18/libf2c/sysdep1.h
+++ b/pmx-2.6.18/libf2c/sysdep1.h
@@ -10,7 +10,11 @@

 #ifdef __linux__
 #define USE_LARGEFILE
+#ifdef __GLIBC__
 #define OFF_T __off64_t
+#else
+#define OFF_T off_t
+#endif
 #endif

 #ifdef _AIX43

